juriscraper/pacer/rss_feeds.py

```diff
@@ -118,19 +119,26 @@ class PacerRssFeed(DocketReport):
     # We use three simple matches rather than a complex one with three groups.
     document_number_regex = re.compile(r"[\"|\'] ?>(\d+)</a>")
     doc1_url_regex = re.compile(r"href=[\"|\'](.*/doc1/.*)[\"|\']")
+    docs1_url_regex = re.compile(r"href=[\"|\'](.*/docs1/.*)[\"|\']")
     short_desc_regex = re.compile(r"\[(.*?)\]")  # Matches 'foo': [ foo ] (
 
     PATH = "cgi-bin/rss_outside.pl"
 
     CACHE_ATTRS = ["data"]
+    other_appellate_courts = ["cadc", "cafc"]
 
     def __init__(self, court_id):
         super().__init__(court_id)
         self._clear_caches()
         self._data = None
         self.session = Session()
         self.is_valid = True
-
+        self.is_appellate = False
+        if (
+            self.court_id[-1].isdigit()
+            or self.court_id in self.other_appellate_courts
```

Comment:
I think I'd just skip this attribute on the class and do `if ... isidigit() or id in ["cadc", "cafc", "cavc"]`

(Note that CAVC is worth supporting here too.)

Reply:
Great, I've changed it and added `"cavc"`.